pkgs/development/python-modules/sentinels/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "sentinels";
+  version = "1.0.0";
```

```suggestion
  version = "1.0.0";

  disabled = pythonOlder "3.5";
```

see https://github.com/vmalloc/sentinels/blob/develop/setup.py#L8